README.md

```diff
@@ -16,4 +16,13 @@ The `autoscaling-model-server` is a charm bundle that can be installed with:
 
 ```
 juju deploy autoscaling-model-server --trust
+
+# Configure knative-serving with Istio information
+# Name of the Istio ingress gateway
+juju config knative-serving istio.gateway.name="<name of the Istio ingress gateway>"
+juju config istio-pilot default-gateway
```

Comment:
Should those be the other way around? IIUC `juju config istio-pilot default-gateway` gives the `<name of the Istio ingress gateway>`, right?

Reply:
or
```suggestion
juju config knative-serving istio.gateway.name="$(juju config istio-pilot default-gateway)"
```
?